e2e/interchaintestv8/relayer_test.go

```diff
@@ -91,3 +104,133 @@ func (s *RelayerTestSuite) TestRelayerInfo() {
 	})
 }
+
+func (s *RelayerTestSuite) TestRelayToEth() {
+	ctx := context.Background()
+	s.SetupSuite(ctx, operator.ProofTypeGroth16)
+
+	eth, simd := s.ChainA, s.ChainB
+
+	ics26Address := ethcommon.HexToAddress(s.contractAddresses.Ics26Router)
+	ics20Address := ethcommon.HexToAddress(s.contractAddresses.Ics20Transfer)
+	transferAmount := big.NewInt(testvalues.TransferAmount)
+	ethereumUserAddress := crypto.PubkeyToAddress(s.key.PublicKey)
+	cosmosUserWallet := s.UserB
+	cosmosUserAddress := cosmosUserWallet.FormattedAddress()
+	sendMemo := "nonnativesend"
+
+	var (
+		transferCoin sdk.Coin
+		txHash       []byte
+	)
+	s.Require().True(s.Run("Send transfer on Cosmos chain", func() {
+		// We need the timeout to be a whole number of seconds to be received by eth
+		timeout := uint64(time.Now().Add(30 * time.Minute).Unix())
+		transferCoin = sdk.NewCoin(s.ChainB.Config().Denom, sdkmath.NewIntFromBigInt(transferAmount))
+
+		transferPayload := ics20lib.ICS20LibFungibleTokenPacketData{
+			Denom:    transferCoin.Denom,
+			Amount:   transferCoin.Amount.BigInt(),
+			Sender:   cosmosUserAddress,
+			Receiver: strings.ToLower(ethereumUserAddress.Hex()),
+			Memo:     sendMemo,
+		}
+		transferBz, err := ics20lib.EncodeFungibleTokenPacketData(transferPayload)
+		s.Require().NoError(err)
+
+		payload := channeltypesv2.Payload{
+			SourcePort:      transfertypes.PortID,
+			DestinationPort: transfertypes.PortID,
+			Version:         transfertypes.V1,
+			Encoding:        transfertypes.EncodingABI,
+			Value:           transferBz,
+		}
+		msgSendPacket := channeltypesv2.MsgSendPacket{
+			SourceChannel:    ibctesting.FirstChannelID,
+			TimeoutTimestamp: timeout,
+			Payloads: []channeltypesv2.Payload{
+				payload,
+			},
+			Signer: cosmosUserWallet.FormattedAddress(),
+		}
+
+		resp, err := s.BroadcastMessages(ctx, simd, cosmosUserWallet, 200_000, &msgSendPacket)
+		s.Require().NoError(err)
+		s.Require().NotEmpty(resp.TxHash)
+
+		txHash, err = hex.DecodeString(resp.TxHash)
+		s.Require().NoError(err)
+
+		s.Require().True(s.Run("Verify balances on Cosmos chain", func() {
+			// Check the balance of UserB
+			resp, err := e2esuite.GRPCQuery[banktypes.QueryBalanceResponse](ctx, simd, &banktypes.QueryBalanceRequest{
+				Address: cosmosUserAddress,
+				Denom:   transferCoin.Denom,
+			})
+			s.Require().NoError(err)
+			s.Require().NotNil(resp.Balance)
+			s.Require().Equal(testvalues.InitialBalance-testvalues.TransferAmount, resp.Balance.Amount.Int64())
+		}))
+	}))
+
+	var multicallTx []byte
+	s.Require().True(s.Run("Retrieve relay tx to Ethereum", func() {
+		resp, err := s.RelayerClient.RelayByTx(context.Background(), &relayertypes.RelayByTxRequest{
+			SourceTxIds:     [][]byte{txHash},
+			TargetChannelId: s.TendermintLightClientID,
+		})
+		s.Require().NoError(err)
+		s.Require().NotEmpty(resp.Tx)
+		s.Require().Equal(resp.Address, ics26Address.String())
+
+		multicallTx = resp.Tx
```

Comment:
Would we want it to be a multi call tx when it's a single tx? I haven't measured, but I would assume the cost is higher than a direct tx?

Reply:
I think consistency is always good. This is not a usecase we are optimizing for, and the gas cost of using multicall vs standard entry point should be very low, compared to all the other reasons gas will be high in that scenario